techniques/html/H44.html

```diff
@@ -51,69 +51,66 @@
-<li><a href="#examples">Examples</a></li>
-<li><a href="#resources">Related Resources</a></li>
-<li><a href="#related">Related Techniques</a></li>
-<li><a href="#tests">Tests</a></li>
+        <ul><li><a href="#technique">このテクニックについて</a></li>
+<li><a href="#description">解説</a></li>
+<li><a href="#examples">事例</a></li>
+<li><a href="#resources">関連リソース</a></li>
+<li><a href="#related">関連テクニック</a></li>
+<li><a href="#tests">検証</a></li>
 </ul>
       </nav>
     </div>
 </aside>
 
 <main id="main" class="standalone-resource__main">
       
-<h1><span>Technique H44:</span>Using label elements to associate text labels with form controls</h1>
+<h1><span>テクニック H44:</span>テキストラベルとフォームコントロールを関連付けるために、label 要素を使用する</h1>
 
 
 
 
 <section id="technique" class="box">
-	<h2 class="box-h box-h-icon">About this Technique</h2>
+	<h2 class="box-h box-h-icon">このテクニックについて</h2>
 	<div class="box-i">
 
 
   
-  <p>This technique relates to:</p>
+  <p>このテクニックは、次に関連する:</p>
   <ul>
   
     <li>
       
-<a href="https://w3c.github.io/wcag/understanding/non-text-content">1.1.1: Non-text Content</a>
-(<strong>Sufficient</strong>)
+<a href="https://waic.jp/translations/WCAG22/Understanding/non-text-content">1.1.1: 非テキストコンテンツ</a>
+(<strong>十分なテクニック</strong>)
     </li>
   
     <li>
       
-<a href="https://w3c.github.io/wcag/understanding/info-and-relationships">1.3.1: Info and Relationships</a>
-(<strong>Sufficient</strong>
-  when used for making information and relationships conveyed through presentation programmatically
-                     determinable)
+<a href="https://waic.jp/translations/WCAG22/Understanding/info-and-relationships">1.3.1: 情報及び関係性</a>
+(提示を通じて伝えられる情報及び関係性をプログラムによる決定が可能にするために使用される場合に<strong>十分なテクニック</strong>)
     </li>
   
     <li>
       
-<a href="https://w3c.github.io/wcag/understanding/labels-or-instructions">3.3.2: Labels or Instructions</a>
-(<strong>Sufficient</strong>)
+<a href="https://waic.jp/translations/WCAG22/Understanding/labels-or-instructions">3.3.2: ラベル又は説明</a>
+(<strong>十分なテクニック</strong>)
     </li>
   
     <li>
       
-<a href="https://w3c.github.io/wcag/understanding/name-role-value">4.1.2: Name, Role, Value</a>
-(<strong>Sufficient</strong>
-  when used with <a href="../general/G108">G108: Using markup features to expose the name and role, allow user-settable properties to be directly set, and provide notification of changes</a>)
+<a href="https://waic.jp/translations/WCAG22/Understanding/name-role-value">4.1.2: 名前 (name)・役割 (role)・値 (value)</a>
+(<a href="../general/G108">G108: Using markup features to expose the name and role, allow user-settable properties to be directly set, and provide notification of changes</a> と併用する場合に<strong>十分なテクニック</strong>)
     </li>
   
   </ul>
   
 
 
 
-  <p>This technique applies to <abbr title="HyperText Markup Language">HTML</abbr> controls that use external labels.</p>
+  <p>このテクニックは、外部のラベルを使用する <abbr title="HyperText Markup Language">HTML</abbr> コントロールに適用される。</p>
```

Comment:
「HTML コントロール」というイディオムっぽく読めてしまうので、「HTML のコントロール」としてみました。(本当は「HTML のフォームコントロール」という意味かと思いますが、原文では「form」が省かれている、と解釈しました。)
```suggestion
  <p>このテクニックは、外部のラベルを使用する <abbr title="HyperText Markup Language">HTML</abbr> のコントロールに適用される。</p>
```